easybuild/easyconfigs/i/IntelPython/IntelPython-2.7.15-2019.2.066.eb

```diff
@@ -0,0 +1,41 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'Binary'
+
+name = 'IntelPython'
+version = '2.7.15'
+pysver = version[0]
+relver = '2019.2.066'
+versionsuffix = '-%s' % relver
+
+homepage = 'https://software.intel.com/en-us/intel-distribution-for-python'
+
+description = """
+ Intel® Distribution for Python. Powered by Anaconda.
+ Accelerating Python* performance on modern architectures from Intel.
+"""
+
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15293/']
```

Comment:
@JackPerdue This is the download URL for `2019 update 3`, should be `.../15115/`?

Reply:
Okay... just made a guess based on our Intel account...  if you say 15115 then I'll do that